src/encoded/static/components/objectutils.js

```diff
@@ -682,16 +682,58 @@ InternalTags.defaultProps = {
     css: '',
 };
 
+/**
+ * Display image with fallback for non-existent image links
+ */
+export class ImageWithFallback extends React.Component {
+    constructor() {
+        super();
+
+        // Initialize image src and alt tags to be empty to ensure component mounts
+        this.state = {
+            imageUrl: '',
+            imageAlt: '',
+        };
+        this.onError = this.onError.bind(this);
+    }
+
+    // Once the component has mounted, update image src and alt tag
+    componentDidMount() {
```

Comment:
I think you might need to comment about this needing to be done at mount so we don’t accidentally change it later, making this fix ineffective. Normally you’d set state with props in the constructor, and I worry I or someone else will see this code and think, “Oh, this can be done easier in the constructor” and change it, breaking the fix.

Reply:
Good point, hopefully this is clearer!